internal/nexodus/nexodus.go

```diff
@@ -178,7 +178,12 @@ func NewNexodus(ctx context.Context,
 		return nil, err
 	}
 
-	ax.nodePrep()
+	// remove orphaned wg interfaces from previous node joins
+	ax.removeExistingInterface()
+
+	if err := ax.symmetricNatDisco(); err != nil {
+		ax.logger.Debug(err)
```

Comment:
shell if we should throw this as a warning? Because this can eventually lead to wrong peer configuration.

Reply:
@vishnoianil I was on the fence, this helped! The other thing I was thinking is this is the best early warning for outbound UDP connectivity issues. Odds are if this random non-rfc google UDP STUN port is blocked, so is WG.